ftw/subsite/browser/selector.py

```diff
@@ -1,9 +1,11 @@
+from Products.Five.browser.pagetemplatefile import ViewPageTemplateFile
+from ftw.subsite.config import IS_PLONE_5
```

Comment:
You introduce a new way of checking for the Plone version here. Could you please do it consistently within the package? -> https://github.com/4teamwork/ftw.subsite/blob/5eebe02b2f7bc230ec97d60cfb85a64b6d44ed53/ftw/subsite/browser/contact.py#L20

Also, I'd check in `ftw.package.__init__.py` usually. The drawback of that is that it will always be loaded...

Reply:
Well the comment should rather be in `ftw.subsite.config.py`, sorry.

Reply:
Thanks 👍